packages/cli/config/jest.js

```diff
@@ -26,6 +26,9 @@ if (fs.existsSync('jest.config.js')) {
 } else {
   const extraOptions = {
     modulePaths: ['<rootDir>'],
+    moduleNameMapper: {
+      '\\.(css|less|scss|sss|styl)$': require.resolve('jest-css-modules'),
```

Comment:
Was it supposed to be "sass"?

Reply:
Do you mean include sass as well? Then no, it's stored in scss files only afaik